src/components/Comment/Comments.scss

```diff
@@ -0,0 +1,74 @@
+@import "~styles/main";
+
+.comment {
+  @include font(Poppins);
+  margin: 20px auto;
+  width: 60%;
+
+  //shame: this is very hacky method. If found an alternative, please replace
+  .avatar {
+    position: fixed;
+    margin: 0px -60px;
+  }
+
+  &__top {
```

Comment:
There already exists mixin to create the element. Why are you not using that mixin? Also, 'top' as a name does not signify its function, use a better name.